### PR TITLE
Record_authentication_events_via_proxy

### PR DESCRIPTION
**Pull request by ioigoume**
- Enable AuthenticationEvents index,view,add through REST API.Support "
- Proxy login" authentication event type.

### CODE REVIEW

File: app/Lib/enum.php [hunk 1]
```diff
@@ -198,6 +198,7 @@ class AuthenticationEventEnum
 {
   const ApiLogin               = 'AI';
   const RegistryLogin          = 'IN';
+  const ProxyLogin             = 'PI';
```
Comment (NicolasLiampotis) [inline app/Lib/enum.php]: ProxyLogin -> UserLogin = 'UL'

File: app/Lib/lang.php [hunk 2]
```diff
@@ -369,7 +369,8 @@
   
   'en.auth.event' => array(
     AuthenticationEventEnum::ApiLogin      => 'API Login',
-    AuthenticationEventEnum::RegistryLogin => 'Registry Login'
+    AuthenticationEventEnum::RegistryLogin => 'Registry Login',
+    AuthenticationEventEnum::ProxyLogin    => 'Proxy Login'
```
Comment (NicolasLiampotis) [inline app/Lib/lang.php]: ProxyLogin --> UserLogin

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,17 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
+## unreleased
+
+### Added
+
+- Enable AuthenticationEvents index,view,add actions through REST API.
+- Support "Proxy login" authentication event type.
```
Comment (NicolasLiampotis) [inline CHANGELOG.md]: "Proxy Login" --> "User Login"

File: app/Model/AuthenticationEvent.php [hunk 4]
```diff
@@ -47,6 +47,7 @@ class AuthenticationEvent extends AppModel {
     'authentication_event' => array(
       'rule' => array('inList',
                       array(AuthenticationEventEnum::ApiLogin,
+                            AuthenticationEventEnum::ProxyLogin,
```
Comment (NicolasLiampotis) [inline app/Model/AuthenticationEvent.php]: ProxyLogin --> UserLogin

**Review by NicolasLiampotis — CHANGES_REQUESTED** [review]
Change "Proxy Login" to "User Login" (or "User Authentication")

File: app/Lib/enum.php (hunk 1 above)
Comment (ioigoume) [reply app/Lib/enum.php]: Done

File: app/Model/AuthenticationEvent.php (hunk 4 above)
Comment (ioigoume) [reply app/Model/AuthenticationEvent.php]: Done

File: app/Lib/lang.php (hunk 2 above)
Comment (ioigoume) [reply app/Lib/lang.php]: Done

File: CHANGELOG.md (hunk 3 above)
Comment (ioigoume) [reply CHANGELOG.md]: Done

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**